tests/tt_eager/python_api_testing/sweep_tests/tt_lib_ops.py

```diff
@@ -1049,7 +1069,7 @@ def eltwise_unary_div_trunc(
     **kwargs,
 ):
     t0 = setup_tt_tensor(x, device, layout[0], input_mem_config[0], dtype[0])
-    t1 = ttl.tensor.div_trunc(t0, value, output_mem_config=output_mem_config)
+    t1 = ttnn.div_no_nan(t0, value, memory_config=output_mem_config)
```

Comment:
It's not clear to me why we need the div_no_nan instead of the div.  Would you comment?

Reply:
div_no_nan returns 0 instead of NaN values in div op. We implemented it since it is a separate op in Tensorflow ([Link](https://www.tensorflow.org/api_docs/python/tf/math/divide_no_nan))